libs/util/FormatUtils.arucas

```diff
@@ -0,0 +1,102 @@
+fun repeatStr(string, count){
+    retVal = '';
+    while (count > 0){
+        retVal = retVal+ string;
+        count--;
+    }
+    return retVal;
+}
+class FormatUtil{
+    static fun showMap(map){
+        if (FormatUtil.checkArgs(map)){
+            return "" + map;
+        }
+        retVal = '{';
+        foreach ( keys : map.getKeys()){
+            if (map.get(keys).instanceOf(Map.type)){
+                retVal = retVal + '\n' + '"'+ keys + '"'+ " : " + FormatUtil.showMap(map.get(keys), 1) + ",";
+            }
+            else if (map.get(keys).instanceOf(List.type)){
+                retVal = retVal + '\n' +'"' + keys + '"'+ " : " + FormatUtil.showList(map.get(keys),1) + ",";
```

Comment:
mutiple unecessary calls to `map.get(keys)`.
Maps and lists are also not the only collections.
`keys` is a weird name since it's actually a singular `key`
And why is `keys` always in quotes? It's not always a string??